test/Test/RegistryDAO.hs

```diff
@@ -36,11 +40,28 @@ test_RegistryDAO = testGroup "RegistryDAO Tests"
 configBS :: DAO.Config
   (RegistryDaoContractExtra ByteString ByteString)
   (RegistryDaoProposalMetadata ByteString ByteString)
-  Empty
+  (CustomParam ByteString ByteString)
 configBS = config
 
 type Parameter =
-  DAO.Parameter (RegistryDaoProposalMetadata ByteString ByteString) Empty
+  DAO.Parameter (RegistryDaoProposalMetadata ByteString ByteString) (CustomParam ByteString ByteString)
+
+getRegistryValue :: (Monad m) => NettestImpl m -> m ()
+getRegistryValue = uncapsNettest $ do
+  -- To check the view, we put a test value in registry during contract initialization
+  -- so that we don't have to create and execute a proposal to put the value there. Some
+  -- dummy values of proposal key and timestamp is used to create the corresponding registry
+  -- entry.
+  let dummyHash = HashUnsafe ""
+  let testKey = encodeUtf8 @Text "testKey"
+  let testVal = encodeUtf8 @Text "testVal"
+  let registryEntry = RegistryEntry (Just testVal) dummyHash  (Timestamp 0)
```

Comment:
I think you may simply use `"testKey"` and `"testVal"` literals here. But I think it would be best if you used `dummy` instead of `test`.
Also, another nitpick: There are two spaces after `dummyHash`.

Reply:
> I think you may simply use "testKey" and "testVal" literals here. 

Yes, I just wanted to make it explicit that the encoding being used is UTF8.

Reply:
Have fixed it except for the explicit utf8 encoding.